diff --git a/node_modules/@twick/live-player/dist/index.js b/node_modules/@twick/live-player/dist/index.js
index ab5daaf..132b3bf 100644
--- a/node_modules/@twick/live-player/dist/index.js
+++ b/node_modules/@twick/live-player/dist/index.js
@@ -55144,7 +55144,7 @@ const Ev = (r, t) => {
   const t = H5().variables.get("input", null)(), e = H5().variables.get("playerId", null)();
   if (t) if (yield r.add(ce(Q1, { fill: t.backgroundColor ?? hE, size: "100%" })), t.tracks) {
     const s = [];
-    for (const a of t.tracks) switch (a.type) {
+    for (const a of [...t.tracks].reverse()) switch (a.type) {
       case r5.VIDEO:
         s.push(K$({ view: r, track: a }));
         break;
diff --git a/node_modules/@twick/live-player/dist/index.mjs b/node_modules/@twick/live-player/dist/index.mjs
index dbcabe7..90c7df4 100644
--- a/node_modules/@twick/live-player/dist/index.mjs
+++ b/node_modules/@twick/live-player/dist/index.mjs
@@ -55142,7 +55142,7 @@ const Ev = (r, t) => {
   const t = H5().variables.get("input", null)(), e = H5().variables.get("playerId", null)();
   if (t) if (yield r.add(ce(Q1, { fill: t.backgroundColor ?? hE, size: "100%" })), t.tracks) {
     const s = [];
-    for (const a of t.tracks) switch (a.type) {
+    for (const a of [...t.tracks].reverse()) switch (a.type) {
       case r5.VIDEO:
         s.push(K$({ view: r, track: a }));
         break;
